Formalist/FormNumberFormatter.swift

```diff
@@ -6,28 +6,22 @@
 //  Copyright © 2017 Seed Platform, Inc. All rights reserved.
 //
 
-public enum FormNumberType {
-    case EIN
-    case SSN
-    case creditCard
```

Comment:
Chatted with @louoso, sounds like there is different type of credit cards, let's use generic pattern that use can set for now. 

![pasted image at 2017_07_25 03_38 pm](https://user-images.githubusercontent.com/24255417/28592859-34b23202-713f-11e7-911b-28de48075905.png)